modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -1,7 +1,7 @@
 = Automatic Failover
 :description: A node or group can be failed over automatically when it either becomes unresponsive or experiences continuous disk-access problems.
```

Comment:
Change to: One or more nodes, or a server group, can be failed over....

Reply:
One or more nodes can be failed over automatically when the nodes become unresponsive or experience continuous disk problems. This is done as long as data safety is maintained i.e. no data will be lost.